examples/nemo/nemo.yaml

```diff
@@ -0,0 +1,59 @@
+# Distributed training a BERT model with Nvidia NeMo
+#
+# Finetunes a BERT-like model on the GLUE CoLA task. Uses the NeMo toolkit
+# to train across multiple nodes, each node having a V100 GPU.
+#
+# Usage:
+#   sky launch -c nemo nemo.yaml
+#   sky down nemo
+
+resources:
+  accelerators: V100:1
+
+num_nodes: 2
+
+setup: |
+  conda activate nemo
+  if [ $? -eq 0 ]; then
+      echo "conda env exists"
+  else
+      conda create -y --name nemo python==3.10.12
```

Comment:
I thought `conda` should only accept single `=`? Also, is the minor version required?

Reply:
Ah, this is from NeMo's [official install instructions](https://github.com/NVIDIA/NeMo#conda), but if you'd like to use just `python=3.10`, I can change it.

Reply:
Ahh, I see. I remembered `==` was not supported by conda, but I think it is fine to follow their official instruction if that works. 
(The conda's [doc](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html#installing-a-different-version-of-python) mentions `python=3.8`)